cmd/util.go

```diff
@@ -346,6 +346,9 @@ func getVersionFromKubectlOutput(s string) string {
 	serverVersionRe := regexp.MustCompile(`Server Version: v(\d+.\d+)`)
 	subs := serverVersionRe.FindStringSubmatch(s)
 	if len(subs) < 2 {
+		if strings.Contains(s, "The connection to the server") {
+			continueWithError(nil, `Warning: Kubernetes version was not auto-detected because kubectl could not connect to the Kubernetes server. This may be because the kubeconfig information is missing or has credentials that do not match the server.`)
```

Comment:
I think that it will be better to just print it to stderr since we are refactoring continueWithError function https://github.com/aquasecurity/kube-bench/pull/630

Reply:
good point!